codegen/templates/workflow.tmpl

```diff
@@ -0,0 +1,247 @@
+{{/* template to render gateway workflow interface code */ -}}
+{{- $instance := .Instance }}
+package workflow
+
+{{- $reqHeaderMap := .ReqHeaders }}
+{{- $reqHeaderMapKeys := .ReqHeadersKeys }}
+{{- $reqHeaderRequiredKeys := .ReqRequiredHeadersKeys }}
+{{- $resHeaderMap := .ResHeaders }}
+{{- $resHeaderMapKeys := .ResHeadersKeys }}
+{{- $clientName := title .ClientName }}
+{{- $clientMethodName := title .ClientMethodName }}
+{{- $serviceMethod := printf "%s%s" (title .Method.ThriftService) (title .Method.Name) }}
+{{- $workflowInterface := printf "%sWorkflow" $serviceMethod }}
+{{- $workflowStruct := camel $workflowInterface }}
+
+import (
+	"context"
+
+	zanzibar "github.com/uber/zanzibar/runtime"
+
+	{{range $idx, $pkg := .IncludedPackages -}}
+	{{$pkg.AliasName}} "{{$pkg.PackageName}}"
+	{{end -}}
+
+	{{if .Method.Downstream }}
+	{{- range $idx, $pkg := .Method.Downstream.IncludedPackages -}}
+	{{$file := basePath $pkg.PackageName -}}
+	{{$pkg.AliasName}} "{{$pkg.PackageName}}"
+	{{end}}
+	{{- end}}
+
```

Comment:
I think we missed `zap.logger` here, workflow using `zap.logger` but didn't import in template, I think IDE is smart enough to `auto-import` during `make generate` XD